tests/README.md

```diff
@@ -1,82 +0,0 @@
-Test Levels
```

Comment:
There's https://tickets.puppetlabs.com/browse/WIN-123 for reviewing all the spec/READMEs, just FYI